lib/mix/lib/releases/appups.ex

```diff
@@ -149,13 +153,25 @@ defmodule Mix.Releases.Appup do
   end
 
   # supervisor
-  defp generate_instruction_advanced(m, true, _is_special, _dep_mods), do: {:update, m, :supervisor}
+  defp generate_instruction_advanced(m, true, _is_special, _dep_mods, _), do: {:update, m, :supervisor}
   # special process (i.e. exports code_change/3 or system_code_change/4)
-  defp generate_instruction_advanced(m, _is_sup, true, []),       do: {:update, m, {:advanced, []}}
-  defp generate_instruction_advanced(m, _is_sup, true, dep_mods), do: {:update, m, {:advanced, []}, dep_mods}
+  defp generate_instruction_advanced(m, _is_sup, true, [], false),        do: {:update, m, {:advanced, []}}
```

Comment:
Can you change the formatting to use two lines for any of these on a single line? Example:

```elixir
defp generate_instruction_advanced(m, _is_sup, true, [], false), 
  do: {:update, m, {:advanced, []}}
```